lib/UI/AboutusUI.dart

```diff
@@ -1,66 +1,72 @@
-                fontWeight: FontWeight.bold,
-                fontSize: 18.0,
+        child: new Scaffold(
+            appBar: AppBar(
+              title: Text('About us'),
+              iconTheme: IconThemeData(
+                color: Colors.white,
               ),
+              backgroundColor: Colors.lightBlue[400],
+              elevation: 0.0,
+              centerTitle: true,
             ),
-          ],
-        ),
-        Image.asset("images/snow.png",height: 150,),
-        Padding(padding: EdgeInsets.only(bottom:25.0),
-        child:
-        Text('All information provided is for general use only and is subject to change.',
-            textAlign: TextAlign.center,
-            style: TextStyle(
-                fontSize: 12.0,
-              ),),
+            body: SafeArea(
+              child: Container(
+                color: Colors.white,
+                padding: EdgeInsets.only(top: 0.0, left: 8.0, right: 8.0),
+                child: Column(
+                  mainAxisAlignment: MainAxisAlignment.start,
+                  children: [
+                    Column(
+                      crossAxisAlignment: CrossAxisAlignment.center,
+                      children: <Widget>[
+                        Image.asset(
+                          "images/bits.jpeg",
+                          height: 200,
+                          width: 450,
+                        ),
+                        SizedBox(height: 8.0),
+                        Text('Copyright © DaysAtBITS',
+                            style: TextStyle(
+                              fontWeight: FontWeight.bold,
+                            )),
+                        SizedBox(height: 15.0),
+                        Text(
+                          'Welcome to DaysAtBITS',
+                          textAlign: TextAlign.center,
+                          style: TextStyle(
+                            fontWeight: FontWeight.bold,
+                            fontSize: 22.0,
+                          ),
+                        ),
+                        SizedBox(height: 5.0),
+                        Text(
+                          'Your number one source for all BITS Hyderabad Anoucements. We’re dedicated to providing you, up to Date updates about everything that’s happening in our campus. The app is under development,\n\n New updates coming soon!',
+                          textAlign: TextAlign.center,
+                          style: TextStyle(
+                            fontWeight: FontWeight.w500,
+                            fontSize: 18.0,
+                          ),
+                        ),
+                      ],
+                    ),
+                    Padding(
+                      padding: EdgeInsets.only(bottom: 10.0),
+                      child: Text(
+                        '\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\n\nAll information provided is for general use only and is subject to change.',
```

Comment:
bro this is not the correct way to show info at the end of a page. If someone's screensize is small then he would get an outOfArea error as the screen isnt scroable. And if the screensize is large then he would see the information somewhere between and not at the end of the page Change this. use a widget which sits at the bottom of the page.

Reply:
delete and correct it please

Reply:
mai delete nhi kr sakta na teri branch ko

Reply:
are branch nahi, add krle yeh then edit kr lena, wont that work?